### PR TITLE
WIP: Fix latest lint reports due to merges

### PR DESCRIPTION
**Pull request by saschagrunert**
TODO: I have to verify why the CI does not report the issues reported locally on my side.

Refers to: https://github.com/kubernetes/repo-infra/pull/169, https://github.com/kubernetes/test-infra/pull/15613

### CODE REVIEW

File: cmd/krel/cmd/push.go [hunk 1]
```diff
@@ -199,6 +199,7 @@ func runPushBuild(opts *pushBuildOptions) error {
 	}
 
 	if opts.versionSuffix != "" {
+		// TODO: what should we do with `latest` afterwards?
 		latest += "-" + opts.versionSuffix
```
Comment (saschagrunert) [inline cmd/krel/cmd/push.go]: The golangci-lint target should throw an error here, which works locally but not inside the test infrastructure.

**Comment by saschagrunert** [comment]
I think that we cannot build everything correctly because of some cgo dependencies missing in the container:
```
Verifying golangci-lint...
level=warning msg="[runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/sandbox/processwrapper-sandbox/214/execroot/io_k8s_release/bazel-out/k8-fastbuild/bin/external/io_k8s_repo_i
nfra/hack/verify-golangci-lint.runfiles/io_k8s_release/cmd/blocking-testgrid-tests/main.go:24:2: could not import net/http (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/net/http/client.go:13:2: could not import crypto/tls (/root/.cach
e/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/crypto/tls/cipher_suites.go:16:2: could not import crypto/x509 (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/crypto/x509/verify.go:11:2: could not import net (/r
oot/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/net/cgo_linux.go:12:8: could not import C (cgo preprocessing failed))))) /root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/sandbox/processwrapper-sandbox/214/execroot/io_k8s_rel
ease/bazel-out/k8-fastbuild/bin/external/io_k8s_repo_infra/hack/verify-golangci-lint.runfiles/io_k8s_release/cmd/blocking-testgrid-tests/main.go:29:2: could not import k8s.io/test-infra/testgrid/config (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/sandbo
x/processwrapper-sandbox/214/execroot/io_k8s_release/_tmp/ca3cea22f615a4ff022845a3169046b1/home/go/pkg/mod/k8s.io/test-infra@v0.0.0-20190829230513-7ef687d80d22/testgrid/config/config.go:27:2: could not import cloud.google.com/go/storage (/root/.cache/bazel/_bazel_root/1cce
3e60ac2d51100ef178d6509a4972/sandbox/processwrapper-sandbox/214/execroot/io_k8s_release/_tmp/ca3cea22f615a4ff022845a3169046b1/home/go/pkg/mod/cloud.google.com/go@v0.38.0/storage/acl.go:19:2: could not import net/http (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6
509a4972/external/go_sdk/src/net/http/client.go:13:2: could not import crypto/tls (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/crypto/tls/cipher_suites.go:16:2: could not import crypto/x509 (/root/.cache/bazel/_bazel_root/1cce3e60ac2
d51100ef178d6509a4972/external/go_sdk/src/crypto/x509/verify.go:11:2: could not import net (/root/.cache/bazel/_bazel_root/1cce3e60ac2d51100ef178d6509a4972/external/go_sdk/src/net/cgo_linux.go:12:8: could not import C (cgo preprocessing failed)))))))]"
level=warning msg="[runner] Can't run linter goanalysis_metalinter: assign: failed prerequisites: inspect@k8s.io/release/cmd/blocking-testgrid-tests"
```

If I install `build-essential` manually then the lint runs perfectly.

**Comment by justaugustus** [comment]
Superseded by https://github.com/kubernetes/release/pull/1004.
/close

**Pull request closed without merging**